MaxText/benchmark_chunked_prefill.py

```diff
@@ -31,7 +31,7 @@
 _BENCHMARK_ITERS = 5
 
 
-def main(argv: Sequence[str]) -> None:
+def main(argv: Union[Sequence[str], Tuple[Any, ...]]) -> None:
```

Comment:
Why is this change being done? is there a called of main that supplies a tupple ?

Reply:
There was a lint error because a `main` was having multiple args (tuple); can't see it anymore this must be old so reverting.